File: 04_TSBackend/server/src/utils/dbConnect.ts
import mongoose from "mongoose"
import config from "config"

const DB_URL: string = config.get("DB_URL")

async function DbConnect() {
    try {

        await mongoose.connect(DB_URL)
        console.log("db connected✅");
        
        
    } catch (error) {
        console.log(error);
        
    }
    
}

export default DbConnect()

File: 09Backend_Practice/server/internal/utils/dbConnect.go
package utils

import (
	"log"

	"github.com/AbdulRahman-04/FullStackWebDev_2026/09Backend_Practice/server/internal/config"
	"gorm.io/driver/postgres"
	"gorm.io/gorm"
)

var PostgresDB *gorm.DB

func ConnectPostgres() {
	db, err := gorm.Open(postgres.Open(config.AppConfig.DB_URL), &gorm.Config{})
	if err != nil {
		log.Fatalf("Couldn't connect postgres")
	}

	PostgresDB = db
	log.Printf("PostgreSQL Connected✅")
}